app/controllers/application_controller.rb

```diff
@@ -207,7 +207,7 @@ def ensure_consent_given
   def ensure_user_belongs_to_community
     return unless @current_user
 
-    if !@current_user.is_admin? && @current_user.accepted_community != @current_community
+    if !@current_user.has_admin_rights? && @current_user.accepted_community != @current_community
```

Comment:
@rap1ds there seems to be and extra check after Hydra, called `ensure_user_belongs_to_community` that logs the user out in case they don't have an approved membership to the community. Now my attempt is to allow admin to use admin panel before having "a fully approved memembership = confirmed email" and this check always threw the admin out. So I extended this exception to cover normal admins in addition to super admins.

I thought that this shouldn't cause risk to the original purpose.  But actually good if you check as you know Hydra bit better, if this could cause any trouble? Could some very old session now be able to hang around in a marketplace they don't belong? (anyway there is a check that should confirm the person has membership with admin status in this community)

Also the comment few lines above suggests that the whole check should be removed at some point. But how can we be sure there are no more old sessions around..?